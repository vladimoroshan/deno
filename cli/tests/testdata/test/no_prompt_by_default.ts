Deno.test("no prompt", () => {
  Deno.readTextFile("./some_file.txt");
});
