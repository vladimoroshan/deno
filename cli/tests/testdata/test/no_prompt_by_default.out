running 1 test from ./test/no_prompt_by_default.ts
no prompt ... FAILED ([WILDCARD]ms)

failures:

no prompt => ./test/no_prompt_by_default.ts:[WILDCARD]
error: PermissionDenied: Requires read access to "./some_file.txt", run again with the --allow-read flag
[WILDCARD]

failures:

no prompt => ./test/no_prompt_by_default.ts:[WILDCARD]

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out ([WILDCARD]ms)

error: Test failed
